### DISCUSSION TITLE
Cant find Avalonia Assembly...

### DISCUSSION THREAD

**Discussion by CosminCAS123** [AvaloniaUI/Avalonia#12885] [Q&A]
I want to bind my view's event to a command in the viewmodel.For that I found the Avalonia.xaml.interactions and Avalonia.xaml.behaviors packages.After installing them from my nuget package manager, they werent showing up in my xaml.
I cloned a git repository with a sample project using them to see if i am missing anything.This one : [sample](https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/tree/master/samples)
To my surprise.they didnt even have the avalonia.xaml.behaviors/interactivity from nuget manager. They just have this:
![fromsample](https://github.com/AvaloniaUI/Avalonia/assets/79158479/caf05a10-cfd8-45ec-b6f4-c6d061a3bc97)
and somehow,in their xaml they are able to do :
```
  xmlns:i ="clr-namespace:Avalonia.Xaml.Interactions.Core;assembly=Avalonia.Xaml.Interactions"
    xmlns:iac="clr-namespace:Avalonia.Xaml.Interactions.Custom;assembly=Avalonia.Xaml.Interactions.Custom"
```
While I can't find them.I now have the exact same packages installed as them and all I get is this:
![myproject](https://github.com/AvaloniaUI/Avalonia/assets/79158479/56d354e8-f307-462f-80b7-67cd5b2b27da)
help?

**Comment by jp2masa** [comment]
> I cloned a git repository with a [sample](https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/tree/master/samples) project using them to see if i am missing anything.This one : sample
To my surprise.they didnt even have the avalonia.xaml.behaviors/interactivity from nuget manager. They just have this:

That's because the samples are using project references.

> While I can't find them.I now have the exact same packages installed as them and all I get is this:

May be some bug in intellisense. Did you try using it and compiling?
